src/App.js

```diff
@@ -1,8 +1,10 @@
-function App() {
-  return (
-    <>
-    </>
-  );
-}
+import { HashRouter, Switch, Route, Redirect } from "react-router-dom";
```

Comment:
It is good practice to import elements that we use. In this case they are imported  `Switch, Route, Redirect` but not used. If they are needed we will just add them in the future.

Reply:
Yes, I have added them intentionally but of course at the moment they are not needed anyway 😄